document/core/exec/instructions.rst

```diff
@@ -464,6 +998,98 @@ Memory Instructions
    \end{array}
 
 
+.. _exec-load-extend:
+.. _exec-load-splat:
+
+:math:`\V128\K{.}\LOAD{M}\K{x}L\_\sx~\memarg` and :math:`\V128\K{.}\LOAD{N}\K{\_splat}~\memarg`
```

Comment:
It feels like these should be split into separate definitions, IMO

Reply:
Yea good point, I'll split them up.